src/server/standards/stu3/base/resources/Appointment.txt

```diff
@@ -0,0 +1,65 @@
+Main
```

Comment:
We should probably move these txt files into the generator project rather than core.